### PR TITLE
PRINE-210-fixed-displaying-correct-start-time-of-reservations

### PR DESCRIPTION
**Pull request by M16**

### CODE REVIEW

File: src/app/components/admin/AdminFacilityTable.vue [hunk 1]
```diff
@@ -211,6 +212,14 @@ export default defineComponent({
       tabIndex.value = index
     }
 
+    function startedAt(booking: BookingModel) {
+      if (booking.state == BookingState.Reserved && booking.reservedAt !== null) {
+        return new Date(booking.reservedAt.toString())
```
Comment (prine) [inline src/app/components/admin/AdminFacilityTable.vue:217]: This makes no sense. It is already a Date you make it a string and then a Date again? 😄

File: src/app/components/admin/AdminFacilityTable.vue [hunk 2]
```diff
@@ -211,6 +212,14 @@ export default defineComponent({
...
+      } else if (booking.startedAt !== null) {
+        return new Date(booking.startedAt.toString())
```
Comment (prine) [inline src/app/components/admin/AdminFacilityTable.vue:219]: This makes no sense. It is already a Date you make it a string and then a Date again? 😄

File: src/app/components/admin/AdminFacilityTable.vue (hunk 1 above)
Comment (M16) [reply src/app/components/admin/AdminFacilityTable.vue:217]: It's just a Date convertion from **Firebase date format** to **JS date format.**

File: src/app/components/admin/AdminFacilityTable.vue (hunk 2 above)
Comment (M16) [reply src/app/components/admin/AdminFacilityTable.vue:219]: It's just a Date convertion from **Firebase date format** to **JS date format.**

**Pull request merged**